Source/WPEFramework/PluginServer.cpp

```diff
@@ -680,11 +683,24 @@ namespace PluginHost
                 result = Core::ERROR_BAD_REQUEST;
             }
             else {
-                #ifdef HIBERNATE_SUPPORT_ENABLED
```

Comment:
@adrianM27 sorry for the late remark but for Thunder we have coding guidelines. IN these coding guidelines it is a rule, for reason of maintainability,  that it is only allowed for 1 return in a method. 
Could we reschuffle this method and the next a bit to only have 1 return statement in the method ? Thanks!

Reply:
Not a problem, aligned per your suggestion